pkgs/development/compilers/ispc/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch
-, cmake, which, m4, python3, bison, flex, llvmPackages, ncurses
+, cmake, which, m4, python3, bison, flex, llvmPackages, ncurses, xcode
```

Comment:
Maybe https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/xcode/sdk-pkgs.nix would be enough, but it seems sort of hardcoded for iOS? 
@matthewbauer